user-docs/documentation/docs/how-to-guides/3-migration-guides/1-yarn-to-pnpm.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 1
+---
+
+# From Yarn to Pnpm
+
+[Yarn](https://yarnpkg.com/) and [Pnpm](https://pnpm.io/) are both package managers.
+
+## Why?
+
+In order to choose between these two alternatives, we have checked the following criteria:
```

Comment:
You can reuse the arguments I pointed out in this issue https://github.com/swarmion/swarmion/issues/261#issue-1367778774